modules/core/react/lib/InputProvider.tsx

```diff
@@ -215,7 +215,7 @@ export default class InputProvider extends React.Component<InputProviderProps> {
     this.enableListeners(false);
   }
 
-  enableListeners(enable: boolean) {
+  enableListeners = (enable: boolean) => {
```

Comment:
I got Typescript errors with out this telling me that use of `this` within this function was set to global... I'm not sure how things worked... Perhaps since there's only meant to be one on the page? I have no idea how this wasn't caught before...

Reply:
Odd... If it's a class function, shouldn't `this` be correct? We should do this to all the functions if it's necessary